articles/quasar-googleauth-with-firebase/index.md

```diff
@@ -0,0 +1,649 @@
+  },
+  methods: {
+    resetPassword () {
+      // firebase reset password
+    }
+  }
+}
+```
+
+This component is imported to the `AuthComponent.vue` which has a form for password resetting.
+
+By now, everything is not displayed correctly yet. Navigate to the `pages` folder open `Auth.vue ` and paste the code below:
+
+```HTML
+<template>
+  <q-page class="flex q-pa-md">
+    <q-card class="full-width">
+      <q-tabs
+        v-model="tab"
+        dense
+        class="text-grey"
+        active-color="primary"
+        indicator-color="primary"
+        align="justify"
+        narrow-indicator
+      >
+        <q-tab name="login" label="Login" />
+        <q-tab name="register" label="Register" />
+      </q-tabs>
+
+
+      <q-tab-panels v-model="tab" animated>
+        <q-tab-panel name="login">
+          <AuthComponent :tab="tab" />
+        </q-tab-panel>
+
+        <q-tab-panel name="register">
+          <AuthComponent :tab="tab"/>
+        </q-tab-panel>
+      </q-tab-panels>
+    </q-card>
+  </q-page>
+</template>
+```
+
+The `AuthComponent` above is imported to the `pages/Auth.vue` with props for register and login tabs. In the `<script>` tag paste the following code:
+
+```JavaScript
+import AuthComponent from "components/AuthComponent";
+export default {
+  components: { AuthComponent },
+  data () {
+    return {
+      tab: 'login'
+    }
+  }
+}
+```
+
+### Add Firebase to our application
+
+To add firebase to our app, ensure the following is done correctly:
+
+1. Visit [firebase console](https://console.firebase.google.com/u/0/).
+2. Click `Add Project` to create a firebase project to connect to our app. Name it `quasar-google-auth` and click `Continue`.
+3. Disable Google Analytics for this project and click `Continue` to create our project.
+4. After the project is ready click `Continue`. It will take you to the project overview. Click the `web` icon, register an app name and name it `quasar-firebase-googleauth`, and click `Register App`.
+5. You'll be taken to `Step 2 - Add Firebase SDK` and copy the whole script.
+
+We're going to install firebase and create a boot file to initialize firebase before our app runs.
+
+Go back to our project, open the terminal, and run the below commands:
+
+```bash
+yarn add firebase
+quasar new boot firebase
+```
+
+Navigate to `quasar.conf.js` file and search for `boot`. It should be an empty array. Add the `firebase.js` boot file in it as indicated below.
+
+```JavaScript
+boot: ['firebase'],
+```
+
+Navigate to the `boot` folder and open the `firebase.js` file. 
+
+Paste the script you copied from the previous step for the firebase SDK. `firebase.js` file structure should resemble the one below.
+
+```JavaScript
+import firebase from "firebase";
+
+const firebaseConfig = {
+    apiKey: "",
+    authDomain: "",
+    projectId: "",
+    storageBucket: "",
+    messagingSenderId: "",
+    appId: ""
+  };
+  firebase.initializeApp(firebaseConfig);
+
+export default firebase
+```
```

Comment:
Hi, you didn't mention what to do with the existing code in this file. it's unclear.

![image](https://user-images.githubusercontent.com/45763204/124802842-06e8ff00-df76-11eb-81a7-e25f6cb5ae8d.png)

Reply:
@EspiraMarvin Please rectify this too.

Reply:
cool.